test/misc.jl

```diff
@@ -28,6 +28,127 @@ let bt = backtrace()
     end
 end
 
+module Foo
```

Comment:
use a more specific name for this or it'll likely fail if run in the same worker as the compile test (e.g. `make testall1`)